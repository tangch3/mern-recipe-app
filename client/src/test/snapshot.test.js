import React from 'react';
import Pagefortesting from '../pages/pagefortesting.js';
import renderer from 'react-test-renderer';

/*
 * To perform a snapshot test, we will construct a tree variable that contains the DOM tree of the rendered component <App /> which is in JSON format.
 * expect(tree).toMatchSnapshot() will create a snapshot of which will be saved.
 * In order to return a JSON object of the rendered DOM tree as a snapshot, the .toJSON() method is used.
 */

test('Renders without crashing?', () => {
  const tree = renderer.create(<Pagefortesting />).toJSON();
  expect(tree).toMatchSnapshot();
});