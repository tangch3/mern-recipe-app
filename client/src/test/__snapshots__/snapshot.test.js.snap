// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders without crashing? 1`] = `
<div>
  <h1>
    Page for Testing Snpshot
  </h1>
</div>
`;
